tests/rt.sh

```diff
@@ -421,6 +421,7 @@ else
 fi
 
 INPUTDATA_ROOT=${INPUTDATA_ROOT:-$DISKNM/NEMSfv3gfs/input-data-20210528}
+INPUTDATA_ROOT_GOCART=/scratch1/NCEPDEV/nems/Raffaele.Montuoro/emc.nemspara/RT/GOCART/input-data-20210614
```

Comment:
No personal directories. This data must be moved under $INPUTDATA_ROOT

Reply:
Updated to reflect shared location (see commit ecd48e2fde649403db8a32763f53c067a634e614).